### DISCUSSION TITLE
You have weak internet connection or the server has problems. Try to refresh the page

### DISCUSSION THREAD

**Discussion by demosinu** [LiveHelperChat/livehelperchat#1914] [Q&A]
You have weak internet connection or the server has problems. Try to refresh the page . why this error i get.

**Comment by remdex** [comment]
It means something not right with your server or your server can't handle a load/request.

**Reply by alonsoac** [reply]
We see this message every time we load a new page. The pending request gets cancelled because we are moving to another chat or menu item. This confuses users although they get used to it as it happens so often.  I think the error should not show when browser is moving to other page. Maybe only show after 2 request errors.

**Reply by remdex** [reply]
This does not happen here, so it's something with your env... You can see official demo, it does not do that...
